app.js

```diff
@@ -0,0 +1,18 @@
+const form = document.querySelector('form');
+const input = document.querySelector('input');
+
+
+addTask = (e) => {
+    e.preventDefault();
+    const titleTask =  input.value;
+    const ul = document.createElement('ul');
+    const li = document.createElement('li');
+    ul.appendChild(li);
+    document.body.appendChild(ul);
```

Comment:
You should have some kind of container for items and `ul` could to that job really well. Adding items at the end of the body isn't a good idea. You'll have hard time to manipulate those items later on.